syntax = "proto3";

import "google/protobuf/empty.proto";

package viewservice;

message HeartbeatRequest {
  string service_identifier = 1;
}

message HeartbeatResponse {
  string message = 1;
}

service ViewService {
  rpc Heartbeat(HeartbeatRequest) returns (google.protobuf.Empty);
}

